communitiesuk/funding-service-design-frontend
# FS-3634, FS-3635: Cookie banner and settings on cookie page

### Change description
Both tickets covered in this PR. Using javascript to deal with the cookie consent as we don't set cookies if javascript is disabled.



### How to test
1. Use dev tools to make sure 'consentMode' is not present in local storage
2. Load the access funding landing page
3. Should be prompted to accept or reject cookies - local storage will be updated tos tore this decision, and cookies stored or not to respect that choice.
4. Navigate to the main 'Cookies' page - you should be able to update your choice from there.


### Screenshots of UI changes (if applicable)
![Screenshot 2023-11-28 at 13 29 46](https://github.com/communitiesuk/funding-service-design-frontend/assets/1729216/594c50c3-022b-4d87-a9c8-d0d00c815249)
